### PR TITLE
Clean up command grouping in README

### PR DESCRIPTION
**Pull request by wp0pw**
Clean up command grouping in README

### CODE REVIEW

**Review by robinbryce — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ PLATFORM=Darwin
 ARCH=arm64
 VERSION=0.0.1
```
Comment (robinbryce) [inline README.md:42]: Can we update this to the hack version. And save figuring out a dynamic solution for future work ?

**Review by robinbryce — DISMISSED** [review]
Fabtasticus. Thanks!

File: README.md [hunk 2]
```diff
@@ -147,18 +148,20 @@ The same command accepts the result of a DataTrails list events call, e.g.
...
+    DATATRAILS_URL=https://app.datatrails.ai
+ 
+    veracity --data-url $DATATRAILS_URL/verifiabledata --tenant=$PUBLIC_TENANT_ID node --mmrindex 916
 
-# Developer commands
+Above command will output `c3323019fd1d325ac068d203c62007b504c5fa762446a9fe5d88e392ec96914b` which will match the value from the merkle log enty page.
 
-The following sub commands are used in development or by contributors. Or
-currently require an authenticated connection
+# General use commands
```
Comment (rcoxhill-01) [inline README.md:163]: This still contains `A simple first example using `nodescan'` which has been moved to IKWID.

File: README.md (hunk 1 above)
Comment (wp0pw) [reply README.md]: done

**Review by robinbryce — APPROVED** [review]
LGTM

**Pull request merged**